### PR TITLE
add icons fonts and icons.json for adding new fonts

### PR DESCRIPTION
**Pull request by ashishmangla0**
adding  icons fonts

### CODE REVIEW

File: css/pages/faq.css [hunk 1]
```diff
@@ -1,61 +1,119 @@
...
+  padding: 0;
+  text-align: center;
+  line-height: 24px;
+  border-width: 1px;
+}
+.faq--show .faq__text{
+  display: block;
+}
+.faq--show .icon-plus::before{
+  content: "\e900";
```
Comment (tejasberi) [inline css/pages/faq.css:111]: if i was coding for this, then i would just used + and - keys on the keyboard and achieved the look and feel of this icon

File: css/common/main.css [hunk 2]
```diff
@@ -1,11 +1,71 @@
+*, ::after, ::before {
+  box-sizing: border-box;
+}
 body {
   font-family: 'Roboto', sans-serif;
+  margin:0
```
Comment (ankushdharkar) [inline css/common/main.css:6]: Semi-colon missing

File: css/pages/faq.css [hunk 3]
```diff
@@ -1,61 +1,119 @@
...
 }
 
+@media (max-width:575.98px){
+  .faq__btn{
+      flex: 0 0 24px;
+  margin-left: 15px;
```
Comment (ankushdharkar) [inline css/pages/faq.css:117]: Formatting needs fixing

File: css/common/main.css [hunk 4]
```diff
@@ -1,11 +1,71 @@
...
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
 
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```
[the remark this answers is not in the document]
Comment (ashishmangla0) [reply css/common/main.css]: fixed

File: css/common/main.css (hunk 2 above)
Comment (ashishmangla0) [reply css/common/main.css]: fixed

File: css/pages/faq.css (hunk 3 above)
Comment (ashishmangla0) [reply css/pages/faq.css:117]: fixed

**Pull request merged**